src/operator/src/insert.rs

```diff
@@ -109,6 +114,47 @@ impl Inserter {
         Ok(Output::AffectedRows(affected_rows as _))
     }
 
+    /// Handle row inserts request with metric engine.
+    pub async fn handle_metric_row_inserts(
+        &self,
+        mut requests: RowInsertRequests,
+        ctx: QueryContextRef,
+        statement_executor: &StatementExecutor,
+        physical_table: String,
+    ) -> Result<Output> {
+        // remove empty requests
+        requests.inserts.retain(|req| {
+            req.rows
+                .as_ref()
+                .map(|r| !r.rows.is_empty())
+                .unwrap_or_default()
+        });
+        validate_column_count_match(&requests)?;
+
+        // check and create physical table
+        self.create_physical_table_on_demand(&ctx, physical_table.clone(), statement_executor)
```

Comment:
Not sure if we can use a default physical table name for the Prometheus write protocol.  Following the COC(
Convention over Configuration) principle.

Reply:
How about adding an extra config to control if enable metric engine? Run without default physical table looks inconvenient

Reply:
That would be great. Adds `enable_metric_engine` to `prom_store` section.